### PR TITLE
Fix parameter serialization and deserialization

### PR DESCRIPTION
**Pull request by rasmus-kirk**
## Purpose

See #110

### Breaking changes

- The following functions now parse using `json-bigint` meaning that they return bigints instead of numbers _for all numbers no matter size_
    - `deserializeReceiveReturnValue`
    - `deserializeReceiveError`
    - `deserializeInitError`
    - `deserializeTypeValue`

### Added

- All JSON serialization in `serialization.ts` is now handled by `json-bigint` meaning that all functions now correctly handles bigint inputs

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: packages/common/CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,17 @@
...
+
+- The following functions now parse using `json-bigint` meaning that they return bigints instead of numbers _for all numbers no matter size_
+    - `deserializeReceiveReturnValue`
+    - `deserializeReceiveError`
+    - `deserializeInitError`
+    - `deserializeTypeValue`
+
 ### Added
 
+- All JSON serialization in `serialization.ts` is now handled by `json-bigint` meaning that all functions now correctly handles bigint inputs
```
Comment (soerenbf) [inline packages/common/CHANGELOG.md:15]: Please move to `## Unreleased` section.

**Review by soerenbf — COMMENTED** [review]
As this included breaking changes, we should merge this into `release/10` instead.

**Review by shjortConcordium — COMMENTED** [review]
LGTM (if Søren's comments are addressed :upside_down_face:)

File: packages/common/CHANGELOG.md (hunk 1 above)
Comment (rasmus-kirk) [reply packages/common/CHANGELOG.md:15]: Whoops, fixed

**Review by soerenbf — APPROVED** [review]
LGTM

**Pull request merged**